### ISSUE TITLE
Expose 'force' and 'removeVolumes' on DockerRemoveContainer

### ISSUE THREAD

**Issue by cdancy** [bmuschko/gradle-docker-plugin#121]
Expose the properties 'force' and 'removeVolumes' for command DockerRemoveContainer. I can take on this work but would like to get #81 merged first, and then #117 done (which is very easy and straightforward), before getting this going. @bmuschko thoughts?

**Comment by bmuschko** [comment]
I added comments on #81. It's not mergeable yet. You can get started on #117 in parallel.

**Comment by bmuschko** [comment]
Is this something you are still planning to work on?

**Comment by cdancy** [comment]
It is and have already done so with a branched version of your project here! I was waiting for you to merge the other PR before I sent in this one. Now that you've done so I can send this in.

**Comment by cdancy** [comment]
@bmuschko sent in PR and everything looks good.

**Comment by bmuschko** [comment]
Thanks for the pull request. It has been merged.
